start-up/setting-up-reaction-roles.md

```diff
@@ -2,7 +2,7 @@
 
 ## What are reaction roles?
 
-MoonlightBot allows users to assign or remove roles from themselves by reacting to specific messages that server admins set up for. There are several customization options to supercharge your server to the top.
+MoonlightBot allows users to assign/remove roles to/from themselves by reacting to specific messages that server admins set up. There are several customization options to optimize your needs.
 
 ## Preliminary
```

Comment:
Add as a prerequisites that the user needs manage server permission, and to have a message written for the reactions to go to